dotcom-rendering/src/components/InteractiveAtom.tsx

```diff
@@ -47,11 +48,18 @@ export const InteractiveAtom = ({
 }: InteractiveAtomType) => {
 	const { renderingTarget } = useConfig();
 
+	const isApps = renderingTarget === 'Apps';
+
+	const onTouchStart = () => getInteractionClient().disableArticleSwipe(true);
+	const onTouchEnd = () => getInteractionClient().disableArticleSwipe(false);
+
 	return (
 		<div
 			css={[containerStyles, !!isMainMedia && fullHeightStyles]}
 			data-atom-id={id}
 			data-atom-type="interactive"
+			onTouchStart={isApps ? onTouchStart : undefined}
+			onTouchEnd={isApps ? onTouchEnd : undefined}
```

Comment:
This doesn't appear to be wrapped in an Island, so what allows this code run on the client?